src/Pelagos/Bundle/AppBundle/Resources/views/template/UI.html.twig

```diff
@@ -1,6 +1,10 @@
 {% extends "PelagosAppBundle::layout.html.twig" %}
 
 {% block head %}
+    
+    <link rel="stylesheet" href="{{ asset('/build/layout.css') }}">
+    <script src="{{ asset('/build/layout.js') }}"></script>
+
     {{
         add_library (
```

Comment:
Instead of making changes to Twig extensions, shouldn't we just go ahead and remove this?

Reply:
Yes, eventually we will do that. But since I consider this pull request just the "framework", we can make all those changes in each "application" template. In this case I could have deleted this section, because I already modified the file, and this function now does nothing. I'm going to change this file.